velero-2-mixin/mixin.libsonnet

```diff
@@ -0,0 +1,19 @@
+local velerolib = import './main.libsonnet';
+
+local velero =
+  velerolib.new()
+  + velerolib.withConfigMixin(
+    {
+      filteringSelector: 'job=~"integrations/velero"',
```

Comment:
Since this is a public mixin in jsonnet-lib. I am wondering if we should be hardcoding the selector `integrations/velerio` here?

Reply:
Yea i think this is fine, it is how its been done historically.

Reply:
Yes, we didn't pay attention to this. Overall, I think putting less strict filter on the mixin level here would be better, then overriding in integrations.